### PR TITLE
[WX-1184] PostgreSQL Docker Image for Local Cromwell

### PR DESCRIPTION
**Pull request by THWiseman**
This PR includes a shell script and IntelliJ Repo Template that should make it very easy to run Cromwell locally using a PostgreSQL database. 

1.) Have docker running
2.) Run the script. From Cromwell root: `./processes/release_processes/scripts/start_publish_postgres_docker.sh`
3.) Launch Cromwell through IntelliJ, using the `Cromwell server TES Postgres` repo template. 

Open to feedback on where the script should live and what it should be called. I borrowed the naming/location of the equivalent mySQL script, but this new script probably shouldn't be associated with `release_processes` at all. 

This work was done while troubleshooting the issue noted in WX-1184.

### CODE REVIEW

File: processes/release_processes/scripts/start_publish_postgres_docker.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+set -euo pipefail
+
+docker run \
+    --name=cromwell_postgres_test_db \
+    --env POSTGRES_USER=cromwell \
+    --env POSTGRES_PASSWORD=test \
+    --env POSTGRES_DB=cromwell_test \
+    --volume ./postgresql-initdb.d:/docker-entrypoint-initdb.d \
+    --publish 5432:5432 \
+    --rm postgres:11.3
```
Comment (aednichols) [inline processes/release_processes/scripts/start_publish_postgres_docker.sh:17]: `11.3` is from 2019, any reason not to specify the latest floating patch like `11-alpine`?

**Review by aednichols — APPROVED** [review]

File: processes/release_processes/scripts/start_publish_postgres_docker.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# This script will launch a Postgres 11.3 container with parameters matching the expectations of Cromwell's CI configuration
```
Comment (jgainerdewar) [inline processes/release_processes/scripts/start_publish_postgres_docker.sh:1]: The fact that this is in `release_processes` with the `publish` label is sort of an artifact - we happen to use the for-Cromwell-publishing MySQL instance for development, because it's there. This probably makes more sense in a reasonable place in `scripts/` (as would the MySQL files... but that seems out of scope for this PR).

**Review by jgainerdewar — APPROVED** [review]

**Comment by jgainerdewar** [comment]
DBMS is failing for unrelated reasons.
